content/scout/quickstart.md

```diff
@@ -108,126 +83,187 @@ the underlying vulnerable express version to 4.17.3 or later.
+Because policy rules are specific to each organization,
+you must specify which organization's policy you're evaluating against.
+Use the `docker scout config` command to configure your Docker organization.
 
-You can see and share the same vulnerability information about an image and
-the other images in your organization in the [Docker Scout Dashboard](./dashboard.md).
+```console
+$ docker scout config organization <ORG_NAME>
+    ✓ Successfully set organization to <ORG_NAME>
+```
 
-All organization members can see an overview of all their images from integrated container registries,
-and get remediation advice at their fingertips. This helps team members in
-security, compliance, and operations to know what vulnerabilities and issues to focus on.
+Now you can run the `quickview` command to get an overview
+of the compliance status for the image you just built,
+using the default, out-of-the-box policy configuration.
 
-1. Select the **Images** tab on the [Docker Scout Dashboard](https://scout.docker.com).
-2. Select any of the tags under
-   the **Most Recent Image** column, and you can see the same vulnerability
-   information as you saw in Docker Desktop and the Docker CLI and share this link with anyone else in your organization.
+```console
+$ docker scout quickview
+
+...
+Policy status  FAILED  (2/7 policies met, 2 missing data)
+
+  Status │                  Policy                   │           Results
+─────────┼───────────────────────────────────────────┼──────────────────────────────
+  ✓      │ Copyleft licenses                         │    0 packages
+  !      │ Critical vulnerabilities                  │    2C     0H     0M     0L
+  !      │ Default non-root user                     │
+  !      │ Fixable critical and high vulnerabilities │    2C    16H     0M     0L
+  ✓      │ High-profile vulnerabilities              │    0C     0H     0M     0L
+  ?      │ Outdated base images                      │    No data
+  ?      │ Supply chain attestations                 │    No data
+```
 
-> **Tip**
->
-> If you are a member of multiple organizations, make sure you select the
-> correct organization from the drop-down in the top right.
-> ![Screenshot showing organization picker in the Docker Scout dashboard](./images/scout-onboarding-org-picker.png)
-{ .tip }
+Exclamation marks in the status column indicates a violated policy.
+Question marks indicate that there isn't enough metadata to complete the evaluation.
+A check mark indicates compliance.
 
-## Step 7: Compare images
+## Step 6: Improve compliance
 
-Over time as you build and push new tags of images, you can use the Docker Scout
-CLI and Dashboard to compare the changes to vulnerabilities and packages in
-different tags of the same image.
+The output of the `quickview` command shows that there's room for improvement.
+Some of the policies are couldn't evaluate successfully (`No data`)
+because the image lacks provenance and SBOM attestations.
```

Comment:
I wonder if we want to elaborate a little here. `No data` just means that the policy evaluation has insufficient data to return a result. In the case above, both the particular policies require the ability to check for attestations, but support for attestations on local images does not yet exist.

Reply:
I don't think we should go into that level of detail in the quickstart guide. I wanted to show an example evaluation and briefly describe the different cases.

Reply:
Yeah, fine by me. I guess I was more wanting to clarify that it isn't necessarily because the image lacks attestations, it's that attestations aren't available locally for any image.

Reply:
🙏🏻 hoping that changes soon 🤩